eng/Versions.props

```diff
@@ -157,7 +157,7 @@
     <RoslynOptProfRunSettingsGeneratorVersion>1.0.0-beta3.19057.1</RoslynOptProfRunSettingsGeneratorVersion>
     <RoslynToolsLightUpSystemRuntimeLoaderFixedVersion>4.3.0</RoslynToolsLightUpSystemRuntimeLoaderFixedVersion>
     <RoslynMicrosoftVisualStudioExtensionManagerVersion>0.0.4</RoslynMicrosoftVisualStudioExtensionManagerVersion>
-    <StreamJsonRpcVersion>1.3.23</StreamJsonRpcVersion>
+    <StreamJsonRpcVersion>2.0.106-beta</StreamJsonRpcVersion>
```

Comment:
What was the minimum version for VS this is available in? Does this need to go into master-vs-deps?

Reply:
it was available while ago. I think 15.6 or 7 something like that. so we should be fine.

Reply:
1.3 is all that VS had through 16.0-preview.1.  
16.0-preview.2 replaced StreamJson 1.3 with 1.4, and also added 1.5 and 2.0.

Reply:
Why are you depending on `2.0.106-beta` here rather than the stable 2.0 build on nuget.org?